README.md

```diff
@@ -26,14 +26,24 @@ Demonstrate your understanding of this Sprint's concepts by answering the follow
 
 - [ ] What is React JS and what problems does it try and solve? Support your answer with concepts introduced in class and from your personal research on the web.
 
+    - React JS is a javascript library created by Facebook for building user interfaces. One problem that React tries to solve is offloading a lot of the operations a browser would normally have to do by creating a virtual copy of the DOM and only updating the elements on the page that are changed by code. This improves performance considerably.
```

Comment:
You hit the answer right on the head.  The performance using the virtual DOM is the main point.  Reusability via the component structure would be second in my opinion and third would be the ability to write JS with HTML via JSX.